gk/fib.c

```diff
@@ -1664,3 +1665,303 @@ del_fib_entry(const char *ip_prefix, struct gk_config *gk_conf)
 
 	return del_fib_entry_numerical(&prefix_info, gk_conf);
 }
+
+struct gk_fib_ether_entry {
```

Comment:
Since this `struct` is only meant to be used for dumping, shouldn't its name be `gk_fib_ether_dump_entry`?